types/src/on_chain_config/execution_config.rs

```diff
@@ -106,8 +118,8 @@ pub struct ExecutionConfigV3 {
 impl Default for ExecutionConfigV3 {
     fn default() -> Self {
```

Comment:
Is this being used anywhere? If so, better to change them to `default_for_genesis` or `default_if_missing` and get rid of this?

Reply:
Nevermind, just realized, this is called from `default_for_genesis` function, but still may be it will be less confusing to remove `Default` trait for `ExecutionConfigV3` and return the desired value from `default_for_genesis` function..

Reply:
Agreed with this,  maybe even just inline there